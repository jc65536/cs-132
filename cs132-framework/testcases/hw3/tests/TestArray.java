class Main {
	public static void main(String[] a){
        int[] x;
        int[] y;
        int z;
        A b;
        b = new A();
        x = new int[4];
        
        x[3] = 2;
        y = x;
        z = 0;
        System.out.println(x.length);
        System.out.println(y[3]);
        System.out.println(y[2]);
        // System.out.println(x[(0-1)]);
        System.out.println(x[(b.return_negative())]);
        // y = new int[(0-1)];
	}
}
class A
{
    int a;
    public int return_negative()
    {
        return 0-5;
    }
}