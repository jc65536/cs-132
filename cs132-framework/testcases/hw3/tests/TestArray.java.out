4
2
0
--- Error (Explicit) ---
array index out of bounds
